src/main/default/lwc/voteCount/voteCount.html

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="slds-grid">
+    <lightning-icon
```

Comment:
Same as above, would it make sense to use a lightning-button-icon?

At the very lease if you use a div for the label you would need to use a <label> and there is an aria attribute called `aria-labelled-by` to indicate which element labels the button icon for accessibility reasons.

Reply:
I'm not sure if using a lightning-button-icon would make sense here since it's not really a button here, the icon is not clickable nor hover-able. It's just displaying an icon to indicate the voting count but its not a button where the user can click.

Reply:
Yea sorry I wrote that too fast and thought that was another button-icon, ignore this :)